spring-cloud-stream-schema-server/src/test/java/org/springframework/cloud/stream/schema/server/SchemaRegistryServerAvroTests.java

```diff
@@ -44,8 +44,7 @@
  * @author Ilayaperumal Gopinathan
  */
 @RunWith(SpringRunner.class)
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT,
-		properties = {"--spring.main.allow-bean-definition-overriding=true"})
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
```

Comment:
Do we really need WEB in this test case?
Just to be sure that we need such an overhead, especially when we use `@DirtiesContext(classMode = AFTER_EACH_TEST_METHOD)`...